guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -0,0 +1,114 @@
+[id="registering-a-host-to-project-using-the-global-registration-template_{context}"]
+= Registering a Host to {ProjectName} Using the Global Registration Template
+
+Use the following procedure to register a host to {ProjectNameX}.
+
+With this registration method you can register hosts to {Project} by generating a `curl` command on {Project} and running this command on hosts.
+This method uses the global registration template that gives you complete control over the process of the host registration.
+
+ifeval::["{build}" != "satellite"]
+Note that you can extend the parameters by plug-ins.
+For more information, see https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc[How to Create a Plugin].
+endif::[]
+
+.Prerequisites
+* The {Project} user that generates the `curl` command must have the `create_hosts` permission.
+* You must have root privileges on the host that you want to register.
```

Comment:
There is more if you plan to register RHEL hosts: activation key (subscription manager won't be happy without it), also if you plan to use automatic insights registration on that RHEL host, you'll need a rhel repo synced and made available through that activation key so that the host can install the insights-client from it

Reply:
Thank you, I have added those prerequisites